zipkin/include/zipkin/tracer.h

```diff
@@ -50,8 +50,10 @@ typedef std::unique_ptr<Reporter> ReporterPtr;
  * service.
  * @return a Reporter object.
  */
-ReporterPtr makeHttpReporter(const char *collector_host,
-                             uint32_t collector_port);
+ReporterPtr makeHttpReporter(
+    const char *collector_host, uint32_t collector_port,
+    SteadyClock::duration reporting_period = std::chrono::milliseconds{500},
```

Comment:
Are we okay with repeating the defaults here as well as in the Options struct, or do we want to put these somewhere shared?

Reply:
👍 Like the idea of putting defaults somewhere shared.